conjure-java-jackson-serialization/versions.lock

```diff
@@ -92,11 +122,41 @@
         "com.fasterxml.jackson.module:jackson-module-afterburner": {
             "locked": "2.9.5"
         },
+        "com.google.code.findbugs:jsr305": {
+            "locked": "3.0.0",
+            "transitive": [
+                "com.google.guava:guava"
+            ]
+        },
+        "com.google.errorprone:error_prone_annotations": {
+            "locked": "2.1.3",
+            "transitive": [
+                "com.google.guava:guava"
+            ]
+        },
         "com.google.guava:guava": {
-            "locked": "20.0",
+            "locked": "23.6.1-jre",
```

Comment:
I think this might actually be fine now as people that need old compatibility are free to stay on remoting3.  If you want the latest and greatest conjure-java-runtime, then you've got to get onto latest guava too.